ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixClusterManager.java

```diff
@@ -55,12 +54,14 @@
   private final MetricRegistry metricRegistry;
   private final ClusterMapConfig clusterMapConfig;
   private final Map<String, DcZkInfo> dcToDcZkInfo = new HashMap<>();
-  private final Map<String, AmbryPartition> partitionNameToAmbryPartition = new ConcurrentHashMap<>();
-  private final Map<String, AmbryDataNode> instanceNameToAmbryDataNode = new ConcurrentHashMap<>();
-  private final Map<AmbryPartition, Set<AmbryReplica>> ambryPartitionToAmbryReplicas = new ConcurrentHashMap<>();
-  private final Map<AmbryDataNode, Set<AmbryReplica>> ambryDataNodeToAmbryReplicas = new ConcurrentHashMap<>();
-  private final Map<AmbryDataNode, Set<AmbryDisk>> ambryDataNodeToAmbryDisks = new ConcurrentHashMap<>();
-  private final Map<ByteBuffer, AmbryPartition> partitionMap = new ConcurrentHashMap<>();
+  private final ConcurrentHashMap<String, AmbryPartition> partitionNameToAmbryPartition = new ConcurrentHashMap<>();
```

Comment:
yes - just making it explicit because this cannot be changed to a non-concurrent one.